tests/admin_views/tests.py

```diff
@@ -2736,7 +2736,7 @@ def test_list_editable_pagination(self):
         self.assertContains(response, 'Unordered object #3')
         self.assertContains(response, 'Unordered object #2')
         self.assertNotContains(response, 'Unordered object #1')
-        response = self.client.get(reverse('admin:admin_views_unorderedobject_changelist') + '?p=1')
+        response = self.client.get(reverse('admin:admin_views_unorderedobject_changelist') + '?p=2')
```

Comment:
I think backwards compatibility isn't critical here, but a mention in the release notes wouldn't hurt.